cpp/test/core/mdspan_utils.cu

```diff
@@ -247,4 +247,20 @@ void test_const_mdspan()
 
 TEST(MDSpan, ConstMDSpan) { test_const_mdspan(); }
 
+void test_contiguous_predicates()
+{
+  extents<std::int64_t, dynamic_extent, dynamic_extent, dynamic_extent> exts{10, 10, 10};
+  {
+    std::array<std::size_t, 3> strides{128, 32, 8};
```

Comment:
Shouldn't this be `{800, 80, 8}`? I must be really misunderstanding something here